t/unit-tests/u-strcmp-offset.c

```diff
@@ -15,21 +15,32 @@ static void check_strcmp_offset(const char *string1, const char *string2,
 			result > 0 ? 1 :
 			0);
 
-	check_int(result, ==, expect_result);
-	check_uint((uintmax_t)offset, ==, expect_offset);
+	cl_assert_equal_i(result, expect_result);
+	cl_assert_equal_i((uintmax_t)offset, expect_offset);
 }
 
 #define TEST_STRCMP_OFFSET(string1, string2, expect_result, expect_offset) \
-	TEST(check_strcmp_offset(string1, string2, expect_result,          \
-				 expect_offset),                           \
-	     "strcmp_offset(%s, %s) works", #string1, #string2)
+	check_strcmp_offset(string1, string2, expect_result,          \
+				 expect_offset)
```

Comment:
I don't really see any benefit in retaining this macro. Previously we were doing some extra stuff, but now it merely forwards the four arguments to `check_strcmp_offset()`. So I'd drop the macro and instead call the function directly.